alert-common/src/main/java/com/synopsys/integration/alert/common/enumeration/ComponentItemPriority.java

```diff
@@ -23,10 +23,11 @@
 package com.synopsys.integration.alert.common.enumeration;
 
 public enum ComponentItemPriority {
-    CRITICAL,
+    HIGHEST,
     HIGH,
     MEDIUM,
     LOW,
+    LOWEST,
     STANDARD;
```

Comment:
STANDARD is lower than LOWEST?

Reply:
I would say so. I don't think STANDARD is correctly named. Should we change it to UNDEFINED?